README.md

```diff
@@ -181,6 +192,9 @@ Popular command line options are:
 - `--embed-tests` - Use with `rainforest export` to export your tests without extracting the
 steps of an embedded test.
 - `--test-folder /path/to/directory` - Use with `rainforest [new, upload, export]`. If this option is not provided, rainforest-cli will, in the case of 'new' create a directory, or in the case of 'upload' and 'export' use the directory, at the default path `./spec/rainforest/`.
+- `--junit` - Create a junit xml report file with the specified name.  Must be run in foreground mode, or with the report command. Uses the rainforest
```

Comment:
We have another tag used to specify files called `--import-variable-csv-file`, so perhaps `--junit-file` may be more consistent and clear.